### DISCUSSION TITLE
关于 主界面宽/高度 的一点小建议

### DISCUSSION THREAD

**Discussion by TnZzZHlp** [STranslate/STranslate#125] [Ideas]
![image](https://github.com/ZGGSONG/STranslate/assets/98214647/1c6a9d7c-09b7-4225-b014-3ad83f4816e2)
这个选项是否可以做成直接根据拖拉调整主界面的窗口大小并且记住，而不是通过修改这两个选项呢

**Comment by ZGGSONG** [comment]
这个是由于主界面为了实现ui效果移除了windows默认样式，鼠标拖拽修改大小也包含在默认样式里面，我曾经尝试恢复该效果，但是发现默认样式引发了诸多ui问题，卡屏白屏，动画不兼容等情况。综合考虑下才有了现在的方案。

当然除了在设置里面修改该配置以外也可以在主界面通过软件内热键快捷修改(不会保存到配置)具体的可以[查阅文档](https://stranslate.zggsong.com/docs/#软件内快捷键)

**Comment by uniquMonte** [comment]
个人觉得目前的这种调节方式比拖拉更好点，调节成一个比较合适的比例之后，使用起来也稳定些。之前用过其它可以拖拉界面的，确实容易出现各种小问题和不稳定
